src/components/SelectedItemBadge/SelectedItemBadge.tsx

```diff
@@ -36,8 +42,8 @@ export interface Props<PriorityItemValue>
 
 const { block, elem } = bem('SelectedItemBadge', styles);
 
-export const SelectedItemBadge = React.forwardRef<HTMLElement, Props<string>>(
-    (
+export const SelectedItemBadge = React.forwardRef(
+    <PriorityItemValue, ChildrenItemValue>(
```

Comment:
Could please explain why do we need `ChildrenItemValue`?

Reply:
before, we were hardcoding the children type to `string`, which caused typescript errors on the application side since the children in the application had a more complex type. Additionally, the priority type was being incorrectly passed to the children as well, which was not appropriate because the children prop has nothing to do with priority. To address this, I introduced separate types for priority and children, making the component more flexible, type-safe, and reusable, while also avoiding typescript errors on the application side

Reply:
I forgot that we have 2 dropdown. It makes sense.